DataFormats/PatCandidates/src/classes_def_objects.xml

```diff
@@ -286,9 +286,12 @@
   <class name="pat::Conversion"  ClassVersion="11">
    <version ClassVersion="11" checksum="1239840459"/>
   </class>
-
-  <class name="pat::PackedCandidate" ClassVersion="28">
-    <version ClassVersion="28" checksum="1956349111"/>  
+  <class name="pat::PackedCandidate::PackedCovariance" ClassVersion="10">
+   <version ClassVersion="10" checksum="3320406063"/>
```

Comment:
ROOT will start by allocating an array of size 10. Using 3 it only allocates 3.
10 was there as a backwards compatibility attempt when we first stared using ClassVersion. It wasn't a very good choice.